src/main/java/gov/usgs/wma/statistics/logic/MonthlyStatistics.java

```diff
@@ -92,7 +92,7 @@ public boolean monthlyStats(List<S> sortedByValue) {
 				
 				builder.minP50(monthYearlyMedians.get(0).value.toString());
 				builder.maxP50(monthYearlyMedians.get( monthYearlyMedians.size()-1 ).value.toString());
-				builder.sampleCount(monthSamples.size());
+				builder.sampleCount(normalizeMutlipleYearlyValues.size());
```

Comment:
One of the decisions was the change the sample count to the median value count rather than the raw actual count.